edx_event_bus_kafka/config.py

```diff
@@ -0,0 +1,59 @@
+"""
+Configuration loading and validation.
+"""
+
+from typing import Optional
+
+import warnings
+import attr
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from django.conf import settings
+
+
+@attr.s
+class CommonConfig:
+    settings = attr.ib(type=dict)
```

Comment:
This is unused. (Also it looks like you committed a `default.db` file by accident?)